src/Applications/Applications/examples/v1.0-beta/Get-MgApplication.md

```diff
@@ -22,7 +22,49 @@ Get-MgApplication -Filter "AppId eq '39b09640-ec3e-44c9-b3de-f52db4e1cf66'" |
 DisplayName     : Test App
 AppId           : 39b09640-ec3e-44c9-b3de-f52db4e1cf66
 SignInAudience  : AzureADandPersonalMicrosoftAccount
-PublisherDomain : contoso.com
+PublisherDomain : Contoso.com
 ```
 
 This examples gets the application by the specified Id.
+
+### Example 3: Get a count of all applications
+
+```powershell
+Get-MgApplication -ConsistencyLevel eventual -Count appCount
+
+Id                                   DisplayName        AppId                                SignInAudience                     PublisherDomain
+--                                   -----------        -----                                --------------                     ---------------
+4f6f867b-4c34-44b2-9680-fe49d024ce74 New apps           bb6e3bbc-8048-412e-a55f-a6bbd8bac525 AzureADandPersonalMicrosoftAccount Contoso.com
+bce20771-2a69-41e5-a0dd-125ac2fa2708 Example 3 App      ffdf268a-2fe2-49e1-8cd7-66ecb61641ec AzureADandPersonalMicrosoftAccount Contoso.com
+dcaa0c3a-8dfa-4b47-bc04-0edbab42043a Principal-Test App bd38ad43-6c46-4cc6-b65c-a0db533a2a6f AzureADMyOrg                       Contoso.com
+f6b30057-7095-4e2c-89f8-224149f591b7 Testing App        00e80963-9bc0-4147-b9e0-2ba56093e7e6 AzureADandPersonalMicrosoftAccount Contoso.com
+fe4caed6-6182-4aca-b70b-b114c5334a8a New app            641992e9-d176-4aff-a3b6-a867b3ba48c4 AzureADandPersonalMicrosoftAccount Contoso.com
+```
+
+This example gets a list of all applications. $appCount will contain the result set count. This example requires the ConsistencyLevel parameter set to eventual because the Count parameter is in the command. For more information on the use of *ConsistencyLevel* and *Count*, see [Advanced query capabilities on Azure AD directory objects](/graph/aad-advanced-queries).
```

Comment:
Are we going to provide permission information in these examples?

Reply:
I will add the permissions for the cmdlets in this PR. Thanks.